java/code/src/com/redhat/rhn/domain/recurringactions/test/RecurringActionFactoryTest.java

```diff
@@ -23,7 +23,43 @@ public void testListMinionRecurringActions() throws Exception {
         assertEquals(List.of(action), RecurringActionFactory.listMinionRecurringActions(minion.getId()));
     }
 
-    public void testListGroupRecurringActions() {
+    public void testListMinionRecurringActions() throws Exception {
```

Comment:
There is two `testListMinionRecurringActions` tests. We should alter the existing one